server.js

```diff
@@ -6,22 +6,112 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
 mongoose.Promise = Promise;
 
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
 
-// Add middlewares to enable cors and json body parsing
+const listEndpoints = require('express-list-endpoints');
+
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+const { Schema } = mongoose;
+
+const ThoughtSchema = new Schema({
+  message: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 140
+  },
+  heart: {
+    type: Number,
+    default: 0
+  },
+  createdAt: {
+    type: Date,
+    default: () => Date.now()
+  }
+}); 
+
+const Thought = mongoose.model("Thought", ThoughtSchema);
+
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.status(200).send({
+    succes: true,
+    message: "OK",
+    body: {
+      conent: "Happy Thoughs API",
+      endpoints: listEndpoints(app)
```

Comment:
Great job using the listendpoints. Made it very easy to see the endpoints in the browser